region/lowernorfair/west/Screw Attack Room.json

```diff
@@ -611,6 +611,131 @@
       ],
       "endsWithShineCharge": true
     },
+    {
+      "link": [1, 4],
+      "name": "Side Platform Jump with Blue Speed",
+      "entranceCondition": {
+        "comeInWithSidePlatform": {
+          "platforms": [
+            {
+              "minHeight": 1,
+              "maxHeight": 1,
+              "minTiles": 17,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"getBlueSpeed": {
+                  "usedTiles": 15,
+                  "openEnd": 1
+                }},
+                {"or": [
+                  "canMomentumConservingMorph",
+                  {"and": [
+                    "canBeVeryPatient",
+                    "canInsaneJump"
+                  ]}
+                ]},
+                {"heatFrames": 80}
+              ],
+              "note": ["This applies to Warehouse Entrance."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 27.4375,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"getBlueSpeed": {
+                  "usedTiles": 24,
+                  "openEnd": 1
+                }},
+                {"heatFrames": 60}
+              ],
+              "note": ["This applies to Dust Torizo Room."]
+            },
+            {
+              "minHeight": 2,
+              "maxHeight": 2,
+              "minTiles": 27.4375,
+              "speedBooster": true,
+              "obstructions": [[1, 0]],
+              "requires": [
+                {"getBlueSpeed": {
+                  "usedTiles": 15,
+                  "openEnd": 1
+                }},
+                "canLateralMidAirMorph",
```

Comment:
This one is a bit odd to rank.  But I think I'll go with `canTrickyDashJump`.  
It's tighter movement than an airball so there is expertish movement on the morph which may come from the `15` tile shortcharge.  A momentumConservingMorph helped once but not usually.

Basically you need to control your shortcharge and do the stutter+early 2 tap.  It feels like you aim for a low height type jump with the amount of speed you get.

Reply:
Right, there's a dropoff in jump height at extra run speed $3.2, so that's the reason for aiming to get more than that much speed. Updated the note to clarify, and added a `canTrickyDashJump` requirement.